src/vsl/parser/grammar/codeBlock.ne

```diff
@@ -1,5 +1,12 @@
 @include "ws.ne"
 
-CodeBlock[statement] -> seperator:* (delimited[$statement {% id %}, seperator:+] seperator:* {% nth(0) %}):? {% (d, l) => new t.CodeBlock(d[1] || [], l) %}
-seperator -> ";"
-           | "\n"
+CodeBlock[statement]
+   -> seperator:* (delimited[$statement {% id %}, seperator:+] seperator:* {%
+            nth(0)
```

Comment:
This can be shortened to `id`, and I don't think a 2-letter postprocessing rule should have it's own line 😛